### ISSUE TITLE
:scheme display

### ISSUE THREAD

**Issue by devlearner** [nghttp2/nghttp2#557]
Hi Tatsuhiro,

Thank you for the release(s)!  As usual, I pulled the latest to try it out
I found something in the console output of nghttpx:

I run nghttpx with a h2 backend (`-b ... ;;proto=h2`); when I issue this h1 request to nghttpx:
[(handcraft) h1 request (w/ relative url)]
```
GET / HTTP/1.1
Host: nghttp2.org

```

The console output of nghttpx displays the h2 request headers to the backend (as usual):
```
[INFO] shrpx_http2_downstream_connection.cc:447 [DCONN:0x200623c8] HTTP request headers
:method: GET
:scheme: http
:path: /
host: nghttp2.org
x-forwarded-proto: http
via: 1.1 nghttpx

```

However when I issue an h1 request with absolute url to nghttpx:
```
GET http://nghttp2.org/ HTTP/1.1
Host: nghttp2.org

```
The console output of nghttpx shows:
```
[INFO] shrpx_http2_downstream_connection.cc:447 [DCONN:0x20067e88] HTTP request headers
:method: GET
:scheme: http://nghttp2.org/
:path: /
:authority: nghttp2.org/
x-forwarded-proto: http://nghttp2.org/
via: 1.1 nghttpx

```
Note the `:scheme` and `x-forwarded-proto:` headers (p.s. also the trailing slash in `:authority`) displayed above. 
Nonetheless it seems to be simply cosmetic: it seems that the h2 request itself issued to the backend is correct; perhaps will it just be a minor display bug?

(Sorry if it's another false report :) )  could u see if it could be reproduced at your side?

Many thanks! (and thanks for the latest release)

**Comment by tatsuhiro-t** [comment]
Thank you for telling us.  This is a real bug, and your observation is correct, and this only affects logging output.  This bug has been fixed in fe0843b.

**Comment by devlearner** [comment]
Thank you for the quick fix!
